File: examples/python/healthCheck.py
# -*- coding: utf-8 -*-
import requests

# 示例：Health check
# 使用 GET 方法请求 /health
url = "http://localhost:8080/api/v1/health"
try:
    response = requests.get(url)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)

File: examples/js/healthCheck.js

const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：Health check
fetch(`http://localhost:8000/api/v1/health`, {
    method: 'GET',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));
